include/RAJA/Timer.hxx

```diff
@@ -91,7 +91,7 @@ public:
                  + std::chrono::microseconds(tstart.tv_usec);
     auto stop = std::chrono::seconds(tstop.tv_sec)
                 + std::chrono::microseconds(tstop.tv_usec);
-    telapsed += std::chrono::duration<double>(stop - start).count();
+    telapsed += std::chrono::duration<long double>(stop - start).count();
```

Comment:
I'm just trying to get some basic things going right now. I don't know what the requirements are for the range of platforms we want to support going forward (BG/Q was always strange, Windows is a new thing, I haven't used the std::chrono stuff before, ...) I'll add an issue to follow up with this later.